<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewImportSwcFromFile</class>
 <widget class="QDialog" name="ViewImportSwcFromFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViewImportSwcFromFile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select Swc Files:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="SelectFilesBtn">
         <property name="text">
          <string>Select Import Files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SelectFolderBtn">
         <property name="text">
          <string>Select Import Folder</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Attach To Project:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ProjectName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTableWidget" name="SwcFileInfo"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="SummaryLabel">
         <property name="text">
          <string>Summary:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ImportBtn">
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CancelBtn">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
